docusaurus/docs/internal/getting-started/quickstart.md

```diff
@@ -26,3 +26,37 @@ API=https://your-rancher yarn dev
 
 > Note: `API` is the URL of a deployed Rancher environment (backend API)
 
+## Troubleshooting 
+
+If `yarn dev` fails with the following error:
+
+```
+Error: error:0308010C:digital envelope routines::unsupported
+    at new Hash (node:internal/crypto/hash:71:19)
+    at Object.createHash (node:crypto:130:10)
```

Comment:
anything in the stack trace below this line could be removed for clarity (which would also remove host machine root path).

Reply:
Fixed. I still left a few of the bottom lines as people might search for them.